tests/test_butler.py

```diff
@@ -111,7 +111,7 @@ def testExecutionExistence(self):
             with self.subTest(chain=chain):
                 datasets = self._get_datasets_from_chain(chain)
                 for ref in datasets:
-                    self.assertTrue(self.butler.datasetExists(ref, collections=chain))
+                    self.assertTrue(self.butler.exists(ref))
```

Comment:
This is one case where `exists_many()` would have been useful but it's not critical given the number of refs being tested.

Reply:
This is nominally a middleware package so in theory there is no downside to using `_exists_many()` here and fixing the code when we make the interface formally public.

Reply:
I have no strong opinion, either way should work OK.